build_apworld.py

```diff
@@ -0,0 +1,158 @@
+import argparse
+import importlib
+import os
+import sys
+import zipfile
+from pathlib import Path
+from typing import Optional, Type, TypeAlias
+
+import orjson
+
+from worlds import AutoWorldRegister
+from worlds.AutoWorld import World
+from worlds.Files import APWorldContainer
+
+StrPath: TypeAlias = str | os.PathLike[str]
+
+MANIFEST_NAME = "archipelago.json"
+ZIP_EXCLUDE = {
+    ".DS_STORE",
+    ".pytest_cache",
+    "__MACOSX",
+    "__pycache__"
+}
+
+
+def main(
```

Comment:
Just a small thing, I think I'd prefer this function to actually be *called* `build_apworld` as well, I'm not a fan of the `module.main` pattern where what the function does comes from the module name. It kinda prevents doing a `from build_apworld import main` style import

Reply:
That makes sense to me. I agree it's more idiomatic to rename this for better imports. Will make this change :+1: